pkg/tid/cli/kernel.go

```diff
@@ -12,12 +13,15 @@ type TidKernel struct {
 	Backend state.Backend
 	// Factory abstracts the creation of services.
 	Factory util.Factory
+	// Config has all the configurations specified in the config file
+	Config types.TomlConfig
```

Comment:
I'm super anal about this kind of stuff, sorry - but I like these things to be in alphabetical order :joy:, same in the constructor below.